contracts/prize-strategy/PrizeSplit.sol

```diff
@@ -0,0 +1,195 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts-upgradeable/math/SafeMathUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+/**
+  * @title Abstract prize split contract for adding unique award distribution to static addresses. 
+  * @author Kames Geraghty (PoolTogether Inc)
+*/
+abstract contract PrizeSplit is OwnableUpgradeable {
```

Comment:
Just want to ensure the Owner is set correctly here - the Harness calls `__Ownable_Init()` but a MultipleWinners is constructed from the Builder - where is the Owner set?

Reply:
@aodhgan To my knowledge when the MultipleWinners owners is set in the builder it will also be applied to inherited `OwnableUpgradeable` contract. 

https://github.com/pooltogether/pooltogether-pool-contracts/blob/feature/prizesplit/contracts/builders/MultipleWinnersBuilder.sol#L80

Added a temporary unit test in MultipleWinners to verify it's initialized with the correct owner.

![image](https://user-images.githubusercontent.com/3408362/125368942-23978500-e330-11eb-8c67-e44830123ae9.png)